packages/patternfly-4/react-charts/src/components/ChartDonutUtilization/ChartDonutThreshold.tsx

```diff
@@ -426,65 +447,100 @@ export const ChartDonutThreshold: React.FunctionComponent<ChartDonutThresholdPro
+  const getLegendAndSubTitleDy = (dynamicTheme: ChartThemeDefinition, position: string) => {
+    const dynamicWidth = donutWidth - (theme.pie.width - dynamicTheme.pie.width);
+    switch (position) {
+      case 'bottom':
+        return getDonutDy(dynamicTheme) + Math.round((donutWidth - dynamicWidth) / 2);
+      default:
+        return getDonutDy(dynamicTheme);
+    }
+  }
+
+  // Returns the horizontal shift for the donut utilization subtitle
+  const getSubTitleDx = (dynamicTheme: ChartThemeDefinition, position: string) => {
+    const dynamicWidth = donutWidth - (theme.pie.width - dynamicTheme.pie.width);
+    switch (position) {
+      case 'right':
+        return getDonutDx(dynamicTheme, legendPosition) + Math.round((donutWidth - dynamicWidth) / 2);
+      default:
+        return Math.round((donutWidth - dynamicWidth) / 2);
+    }
+  };
+
   // Render dynamic utilization donut cart
   const renderChildren = () =>
     React.Children.toArray(children).map(child => {
-      if (child.props) {
-        const datum = getData([{ ...child.props.data }]);
-        const orientation = child.props.donutOrientation || donutOrientation;
-        const dynamicTheme =
-          child.props.theme ||
-          getDonutThresholdDynamicTheme(child.props.themeColor || themeColor,
-            child.props.themeVariant || themeVariant);
-        return React.cloneElement(child, {
-          donutDx: child.props.donutDx || getDynamicDonutDx(dynamicTheme, orientation),
-          donutDy: child.props.donutDy || getDynamicDonutDy(dynamicTheme),
-          donutHeight: child.props.donutHeight || donutHeight - (theme.pie.height - dynamicTheme.pie.height),
-          donutOrientation: orientation,
-          donutWidth: child.props.donutWidth || donutWidth - (theme.pie.width - dynamicTheme.pie.width),
-          endAngle: child.props.endAngle || 360 * (datum[0]._y ? datum[0]._y / 100 : 100),
-          height: child.props.height || height,
-          showStatic: child.props.showStatic || false,
-          standalone: false,
-          theme: dynamicTheme,
-          width: child.props.width || width
-        });
+      const props = child.props;
+      if (!props) {
+        return child;
       }
-      return child;
+      const datum = getData([props.data]);
+      const legendPos = props.legendPosition || legendPosition;
+      const subTitlePos = props.subTitlePosition || subTitlePosition;
+      const dynamicTheme = getDonutThresholdDynamicTheme(
+        props.themeColor || themeColor, props.themeVariant || themeVariant);
+      return React.cloneElement(child, {
+        donutDx: getDonutDx(dynamicTheme, legendPos),
+        donutDy: getDonutDy(dynamicTheme),
+        donutHeight: donutHeight - (theme.pie.height - dynamicTheme.pie.height),
+        donutWidth:donutWidth - (theme.pie.width - dynamicTheme.pie.width),
+        endAngle: 360 * (datum[0]._y ? datum[0]._y / 100 : 100),
```

Comment:
Not new, but this could result in an end angle value of `36000`:

```suggestion
        endAngle: 360 * (datum[0]._y || 100 / 100),
```

Reply:
That produces some strange results

<img width="230" alt="Screen Shot 2019-06-14 at 11 59 07 AM" src="https://user-images.githubusercontent.com/17481322/59522118-e2f08780-8e9b-11e9-82fe-78818ebdcc4a.png">

Reply:
The end angle should be zero with no value, so looking at this:
```
endAngle: 360 * (datum[0]._y ? datum[0]._y / 100 : 0),
```

Reply:
That could also be simplified to 
```js
endAngle: 360 * (datum[0]._y || 0 / 100),
```

Reply:
That also did not work properly because it's either `datum[0]._y` or `0/100`

Reply:
Oh, it's just an order of ops problem:

```
endAngle: 360 * ((datum[0]._y || 0) / 100),
```
Would probably work, but it's not really important. Just a nit.